br/pkg/task/restore.go

```diff
@@ -377,6 +393,18 @@ func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConf
 		return nil
 	}
 
+	if cfg.WithPlacementPolicy {
```

Comment:
The spec says:

1. BR supports all restore policies when using  `br restore full`;
2. While using `br restore db/table` or [table filter](https://docs.pingcap.com/tidb/stable/use-br-command-line-tool#restore-with-table-filter), The policies associated with the restored table will be restored;

So I think which policy should be restored will be determined by the restore mode and the restored tables/schemas. We only need to set the associative system variable according to this option.

Reply:
good point. I'll update it.